ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -791,6 +800,16 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: e2e-aws-ipi-sdn-mtu-migrate-p3
```

Comment:
We can set cron for this test as ` cron: 0 18 */28 * *` since it is p3.
BTW, better move the test forward (to before line 348) and place it with other aws tests together.